upload/upload.go

```diff
@@ -31,6 +31,13 @@ type uploadData struct {
 	Account string `json:"account_number,omitempty"`
 }
 
+var contentSizeMap map[string]int64
```

Comment:
You might consider building the hash map in main.go and injecting it into the handler.  This would remove the global.  It might make unit testing a bit easier.

Reply:
Also, if you wanted, you could also use `Viper.GetStringMapString()` (https://pkg.go.dev/github.com/spf13/viper#GetStringMapString) to read a  hash map from a config option (env var).  This way you could remove any mention of qpc from your code.  This would also allow you to adjust the maximum upload sizes per service type without recompiling/re-releasing.